src/Particular.Packaging/build/Particular.Packaging.targets

```
@@ -80,7 +80,7 @@
     <WriteLinesToFile File="$(GITHUB_ENV)" Lines="@(MinVerProperties)" ContinueOnError="WarnAndContinue" />
   </Target>
 
-  <Target Name="AddReleaseNotesLink" BeforeTargets="GenerateNuspec">
+  <Target Name="AddReleaseNotesLink" BeforeTargets="GenerateNuspec" DependsOnTargets="GenerateSourceLinkFile;MinVer">
```

The target you were referencing before, `InitializeSourceControlInformationFromSourceControlManager` does seem to be the target that actually sets the value of `ScmRepositoryUrl`, so depending on that instead makes sense to me.

Also, I'd prefer to see the DependsOnTargets first.


```suggestion
  <Target Name="AddReleaseNotesLink" DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager;MinVer" BeforeTargets="GenerateNuspec" >
```